doc/Documentation_Basics.md

```diff
@@ -0,0 +1,175 @@
+---
+title: "Documentation_Basics"
+---
+
+##Installation
+- Java SE Development Kit 17.x.x oder höher benötigt
+- in IDE einbinden
+- libGDX wird verwendet, ist aber schon eingebunden und muss nicht extra installiert werden
+
+#### kurzer Einschub: was ist ein ECS?
+- siehe [readme.md](https://github.com/Programmiermethoden/Dungeon/blob/master/doc/ecs/readme.md)
+
+##Struktur des Repos
+
+Relevante Klassen zum Einstieg:
+Package "starter"
+Game.java (Haupt-Steuerung des Dungeon)
+- beinhaltet main
+- verwaltet die anderen Controller und Systeme
+- alle Systeme befinden sich im „package ecs→ systems“
```

Comment:
die Systeme würde ich hier nicht auflisten sondern den Link zu der bereits existierenden doc. 

Mir wäre hier wichtig das `ECS_System` zu erklären. Also wie steht das in zusammenhang mit Game? WIe wird das in die Loop eingebunden? Wann werden die Sachen aufgerufen?